### ISSUE TITLE
Remove `window.__CI_APP_VERSION__` environment parameter, add `process.env. ARTIFACT_VERSION` instead

### ISSUE THREAD

**Issue by ranyitz** [wix/yoshi#1602]
This is a continuation of the discussion started in this PR #1496

We would like to remove `window.__CI_APP_VERSION__` environment parameter and instead add the `process.env.ARTIFACT_VERSION` which should be aligned with 
https://github.com/wix/yoshi/blob/e28c081225e56ca660942ba5b1d853487ad73a8a/packages/yoshi/config/webpack.config.js#L136

This is a breaking change, and this is why we would like to add it to version 5 of Yoshi.

**Comment by sidoruk-sv** [comment]
@ranyitz am I correct, that this issue is already fixed?
https://github.com/wix/yoshi/blob/master/packages/yoshi-common/src/webpack.config.ts#L727

**Comment by ranyitz** [comment]
@sidoruk-sv You are correct that this was added, but we still need to have this issue for the removal of [process.env.__CI_APP_VERSION__](https://github.com/wix/yoshi/blob/1c0df4d93fd6bfedd9f789c691eff08199df730c/packages/yoshi-common/src/webpack.config.ts#L722-L726) in Yoshi 5.
